docs/data/styles/api/api.md

```diff
@@ -248,7 +248,7 @@ function App() {
   return <StylesProvider jss={jss}>...</StylesProvider>;
 }
 
-ReactDOM.render(<App />, document.querySelector('#app'));
+ReactDOM.createRoot(document.querySelector('#app')).render(<App />);
```

Comment:
Isn't this page about `@mui/styles`? If so then we shouldn't use React 18 APIs since that package isn't compatible with React 18.

Reply:
Good catch @eps1lon!